lib/routes/fcmtokens.js

```diff
@@ -0,0 +1,248 @@
+const router = require('koa-router')()
+const models = require('../models')
+
+router
+  .get('/fcmtokens', async (ctx) => {
+    ctx.body = await models.db.fcmtoken.findAll()
+  })
+  .get('/fcmtokens/:id', async (ctx) => {
+    ctx.body = await models.db.fcmtoken.findById(ctx.params.id)
+  })
+  .get('/fcmtokens/token/:token', async (ctx) => {
+    ctx.body = await models.db.fcmtoken.findOne({where:
+      {fcm_token: ctx.params.token}
+    })
+  })
+  .get('/fcmtokens/participant/:fbid', async (ctx) => {
+    let fbid = ctx.params.fbid
+    await models.db.participant.findOne({
+      where: {fbid}
+    })
+      .then(async (participant) => {
+        if (participant) {
+          ctx.body = await models.db.fcmtoken.findOne({
+            where: {participant_id: participant.id}
+          })
+        } else {
+          ctx.status = 204
+          ctx.body = {'error': {
+            'code': 204,
+            'message': `participant with fbid "${fbid}" does not exist`
+          }}
+        }
+      })
+  })
+  .post('/fcmtokens', async (ctx) => {
+    let fcmTokenVal = ctx.request.body.fcm_token
+    let fbid = ctx.request.body.fbid
+    let participantId = ''
+    let fcmtoken = await models.db.fcmtoken.findOne({where: {fcm_token: fcmTokenVal}})
+    if (fcmtoken) {
```

Comment:
I'm not sure I follow the logic with these deeply nested conditionals. Please add some comments to clarify what each check is for and try to reduce the nesting. Also, the following brace style is preferred:
```
if (condition1) {
  <code1>
} else if (condition2) {
  <code2>
} else {
  <code3>
}
```